pkg/operator/configobservation/controlplanereplicascount/observe_control_plane_replicas_count.go

```diff
@@ -0,0 +1,89 @@
+package controlplanereplicascount
+
+import (
+	"fmt"
+	"github.com/ghodss/yaml"
+
+	"github.com/openshift/cluster-etcd-operator/pkg/operator/configobservation"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+)
+
+var ControlPlaneReplicasCountPath = []string{"controlPlaneReplicasCount"}
```

Comment:
naw, nest under something.  You know this won't be right.

Reply:
Nit: Elsewhere we just use `replicas`, so maybe `controlPlane`: `replicas`? Not sure count adds much does it?